itou/utils/logging.py

```diff
@@ -10,4 +10,7 @@ def json_record(self, message, extra, record):
         for log_key in self.LOG_KEYS_TO_REMOVE:
             if log_key in log_entry_dict:
                 del log_entry_dict[log_key]
+        wsgi_request = self.get_wsgi_request()
+        if wsgi_request is not None and (current_org := getattr(wsgi_request, "current_organization", None)):
```

Comment:
Ce ne sera pas Falsey. Comme tu préfères.
```suggestion
        if wsgi_request and (current_org := getattr(wsgi_request, "current_organization", None)):
```